README.md

```diff
@@ -172,6 +172,17 @@ Call *method* on the SOAP service.
       // soapHeader is the response soap header as a JavaScript object
   })
 ```
+
+A *method* can also be called as a promise.
+
+``` javascript
+  client.MyFunction({name: 'value'}).then(function({result, raw, soapHeader}){
```

Comment:
Can you add an example showing `await` usage too please? In my experience, many developers don't realize it's possible to `await` any function that's returning a promise.

```js
const {result, raw, soapHeader} = client.MyFunction({name: 'value'});
```